src/clincoded/static/components/variant_central/interpretation/segregation/evidenceTable.js

```diff
@@ -116,37 +116,53 @@ let EvidenceTable = createReactClass({
         } else {
             let content = null;
             let help = null;
+            const separator = ', ';
             switch (metadata['_kind_key']) {
                 case 'clinical_lab':
                     content = metadata.lab_name;
-                    help = `Clinvar/GTR LabID: ${metadata.clinvar_gtr_labid}` + (metadata.contact ? `, Contact: ${metadata.contact}` : '');
+                    help = (metadata.clinvar_gtr_labid ? `Clinvar/GTR LabID: ${metadata.clinvar_gtr_labid}` : '') +
+                           (metadata.clinvar_gtr_labid && metadata.clinvar_scv ? `${separator}` : '') +
```

Comment:
```suggestion
                           (metadata.clinvar_gtr_labid && metadata.clinvar_scv ? separator : '') +
```
`separator` is already a string, so you don't need the `${...}` syntax
Same for the other `${separator}` usages in this file

Reply:
Fixed.  Thanks.